code/controllers/subsystems/traumas.dm

```diff
@@ -0,0 +1,125 @@
+
+					   "lizards"   = typecacheof(list(/obj/item/toy/plushie/lizard,/obj/item/weapon/material/hatchet/unathiknife,/obj/item/weapon/rig/unathi,
+												/obj/item/clothing/suit/unathi/robe, /obj/structure/sign/flag/hegemony, /obj/item/stack/material/animalhide/lizard,
+												/obj/item/toy/plushie/lizard)),
+
+					   "skrell" = typecacheof(list(/obj/structure/sign/flag/jargon, /obj/item/clothing/ears/skrell, /obj/item/clothing/suit/space/skrell,
+												 /obj/item/clothing/head/helmet/space/skrell, /obj/item/organ/brain/skrell, /obj/item/organ/eyes/skrell,
+												 /obj/item/organ/heart/skrell, /obj/item/organ/kidneys/skrell, /obj/item/organ/liver/skrell, /obj/item/organ/lungs/skrell,
+												 /obj/item/weapon/reagent_containers/food/snacks/skrellsnacks)),
+
+					   "robots" = typecacheof(list(/obj/item/device/electronic_assembly/drone, /obj/item/weapon/holder/drone, /obj/item/device/mmi/digital,
+												 /obj/item/organ/ipc_tag, /obj/machinery/computer/borgupload, /obj/item/clothing/suit/cyborg_suit,
+												 /obj/machinery/cryopod/robot, /obj/machinery/robotic_fabricator, /obj/effect/decal/cleanable/blood/gibs/robot, /obj/effect/decal/remains/robot,
+												 /obj/item/robot_parts, /obj/item/organ/cell, /obj/item/organ/data, /obj/item/toy/figure/borg)),
+
+					   "nanotrasen" = typecacheof(list(/obj/item/toy/nanotrasenballoon, /obj/item/weapon/soap/nanotrasen, /obj/structure/sign/flag/nanotrasen,
+												 /obj/item/clothing/suit/storage/leather_jacket/nanotrasen, /obj/item/clothing/suit/storage/toggle/brown_jacket/nanotrasen, /obj/item/weapon/storage/toolbox/lunchbox/nt,
+												 /obj/structure/banner, /obj/structure/bed/chair/office/bridge, /obj/item/clothing/head/collectable/captain,
+												 /obj/item/clothing/head/helmet/space/void/captain, /obj/item/clothing/suit/space/void/captain, /obj/item/clothing/suit/storage/hooded/wintercoat/captain,
+												 /obj/item/clothing/under/captainformal, /obj/item/clothing/under/rank/captain, /obj/item/toy/figure/captain, /obj/item/weapon/card/id/captains_spare,
+												 /obj/item/weapon/gun/energy/captain, /obj/item/weapon/stamp/captain, /obj/item/weapon/storage/backpack/captain, /obj/item/clothing/suit/armor/vest/security)),
+
+					   "filth"   = typecacheof(list(/obj/effect/decal/cleanable,/obj/effect/decal/remains,/obj/item/trash, /obj/machinery/portable_atmospherics/hydroponics)),
+
+					   "paranormals"   = typecacheof(list(/obj/item/weapon/book/tome,/obj/item/weapon/nullrod,/obj/structure/constructshell,
+												/obj/structure/cult, /obj/structure/girder/cult, /obj/structure/grille/cult, /obj/effect/forcefield/cult,
+												/obj/effect/gateway, /obj/item/clothing/head/culthood, /obj/item/clothing/head/helmet/space/cult, /obj/item/clothing/shoes/cult,
+												/obj/item/clothing/suit/cultrobes, /obj/item/clothing/suit/space/cult, /obj/item/toy/cultsword, /obj/item/weapon/melee/cultblade,
```

Comment:
Including a toy sword and a bible in this seems silly.

Reply:
Bibles are the very essence of the supernatural.